src/Parameters/CreateMeetingParameters.php

```diff
@@ -526,7 +526,7 @@ public function getHTTPQuery(): string
 
         if ($this->isBreakout()) {
             if ($this->parentMeetingID === null || $this->sequence === null) {
-                trigger_error('Breakout rooms require a parentMeetingID and sequence number.', \E_USER_WARNING);
+                throw new \RuntimeException('Breakout rooms require a parentMeetingID and sequence number.');
```

Comment:
Changed since PHPUnit 10 in future won't be able to expect these anymore and bad practice anyway.